database/buildSchema/13_template_permission.sql

```diff
@@ -20,7 +21,9 @@ CREATE VIEW permissions_all AS (
         template_permission.id AS "templatePermissionId",
         template_permission.stage_number AS "stageNumber",
         template_permission.level_number AS "reviewLevel",
-        template_permission.restrictions AS "templatePermissionRestrictions",
+        template_permission.restrictions AS "restrictions",
+        template_permission.allowed_sections AS "allowedSections",
+        template_permission.can_self_assign AS "canSelfAssign",
```

Comment:
Out of scope, but since we're changing this...

I been wanting to adjust this for a while and put the most relevant (for reading) columns first so I didn't constantly have to scan side-to-side when browsing this View table.

I'm thinking...

```
       "user"."username" AS "username",
        organisation.name AS "orgName",
        "template".code AS "templateCode",
        permission_name.name AS "permissionName",
        template_permission.stage_number AS "stageNumber",
        template_permission.level_number AS "reviewLevel", 
        template_permission.allowed_sections AS "allowedSections",
        template_permission.can_self_assign AS "canSelfAssign",
        template_permission.restrictions AS "restrictions",
        permission_policy.name AS "policyName",
        permission_policy.type AS "permissionType",
        permission_policy.id AS "permissionPolicyId",
        permission_policy.rules AS "permissionPolicyRules",
        permission_name.id AS "permissionNameId",
        template_permission.id AS "templatePermissionId",
        "template".id AS "templateId",
        "user".id AS "userId",
        permission_join.organisation_id AS "orgId"
```

Sound about right?

Reply:
No problem. I'll quickly do it now